### DISCUSSION TITLE
Card: Osama bin Laden

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#10] [Ideas]
- Clarification: What does "at 9/11" mean? When the '9/11 attacks' card is played as an Event, possibly even for Ops, or the beginning of turn 5?
- The beginning of turn 5. As I said, i like the Pre/Post 9/11 wording to keep the theme in-game. ..
- Then I have this suggestion. I'm not completely satisfied with a "continuous effect" that belongs to the previous conditions, then followed by another separate condition. What do you think?
    > **Played in First Epoch**: US removes 1 Influence from one country among the Middle East and Afghanistan.
    > **Continuous effect**: US can conduct Influence Ops in these countries. Cancel the effect at the beginning of the Second Epoch and return this card to the discard pile.
    > **Played in Second Epoch**: US loses 1 VP and removes 1 US Influence from one country among the Middle East and Afghanistan, then return this card to the discard pile.

  Also removed the asterisk since the card always returns to the discard pile. We'll consider the Epoch vs "9/11" wording separately.

**Comment by rodrigoSantamaria** [comment]
Why remove the asterisk? If played by text, this always goes to the removed from the game pile, not the discard pile.
It should say "remove this card from the game" instead of "return this card to the discard pile".

**Comment by RebelusQuo** [comment]
Ah, that changes the cards mechanic. Your original text ended with ".. , then discard this card." which implies the discard pile. New attempt:

> **Played in First Epoch**: US removes 1 Influence from one country among the Middle East and Afghanistan.
> **Played in Second Epoch**: US loses 1 VP and removes 1 US Influence from one country among the Middle East and Afghanistan, \
> then prevent this cards continuous effect.
> **Continuous effect**: US can conduct Influence Ops in these countries. \
> Cancel the effect at the beginning of the Second Epoch.

With the asterisk retained.

**Comment by rodrigoSantamaria** [comment]
Yes. Sorry, this card is complex and my wording was awful.
The original meaning was a bit different: the card has a posterior trigger if played on 1st epoch, making US to resolve the text again, as if in second epoch, before discarding the card, when 2nd epoch begins. 
In the playtests, that may be a little bit too much penalty for US. We can change the trigger not to 2nd epoch, but to the actual play of the 9/11 attacks card by its text.
I don't know if i'm explaining myself or being more confusing.

**Comment by RebelusQuo** [comment]
This is indeed a tricky beast. Here's a new attempt:
> **Played in First Epoch**: US removes 1 Influence from one country among the Middle East and Afghanistan.
> **Played in Second Epoch**: Prevent this cards continuous effect, then perform its triggered effect.
> **Continuous effect**: US can conduct Influence Ops in these countries. Cancel the effect at the beginning of the Second Epoch, then perform its triggered effect.
> **Triggered effect**: US loses 1 VP and removes 1 US Influence from one country among the Middle East and Afghanistan.

**Reply by RebelusQuo** [reply]
Technically it should be "trigger its triggered effect", but let's not give the department of redundancy department too much credit, shall we?

**Reply by rodrigoSantamaria** [reply]
Hehehe, yes, let it be without redundancy.
I think like this is pretty well written. My only concern is if people will understand well that when a card has a continuous or triggered effect, the card remains in play. I guess it's a matter of making it clear in the rulebook, though.

**Reply by RebelusQuo** [reply]
It is definitely the intention to spell these keywords out in the rulebook!
